tools/api-markdown-documenter/src/api-item-transforms/configuration/Transformations.ts

```diff
@@ -59,7 +59,7 @@ export type TransformApiItemWithoutChildren<TApiItem extends ApiItem> = (
  *
  * @public
  */
-export interface ApiItemTransformationOptions {
+export interface ApiItemTransformations {
```

Comment:
Rename required to allows creating separate `Configuration` and `Options` types. But it also aligns this type better with similar "transformations" types used elsewhere in the library (e.g., in the "renderers").